web/packages/teleport/src/lib/tdp/client.ts

```diff
@@ -325,12 +325,12 @@ export default class Client extends EventEmitterWebAuthnSender {
     );
   }
 
-  handleRDPConnectionInitialized(buffer: ArrayBuffer) {
+  handleRDPConnectionActivated(buffer: ArrayBuffer) {
```

Comment:
```suggestion
  handleRdpConnectionActivated(buffer: ArrayBuffer) {
```

Here and below (we don't capitalize initialisms in JS code the way we would in Go)